target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -172,13 +154,10 @@
 
 &esw {
 	mediatek,portmap = <0x3e>;
+	mediatek,portdisable = <0x26>;
 };
 
 &sdhci {
 	status = "okay";
 	mediatek,cd-high;
 };
-
-&wmac {
-	status = "okay";
-};
```

Comment:
That's not a pin change.

Reply:
This was just an obvious doublet in the dts file - so it's just a cosmetic change - i can put it to a separate commit.

> "factory image" typically refers to an OpenWrt image built for flashing over vendor firmware. I suspect you mean the OEM/vendor firmware when you refer to "factory image"?

yes you are right - i'll change this